### ISSUE TITLE
Unmarshaling error when using stringified JSON option (-d)

### ISSUE THREAD

**Issue by fagnercarvalho** [bojand/ghz#255]
**Proto file(s)**
```proto
syntax = "proto3";

package proto;

option go_package="github.com/fagnercarvalho/go-nearby-flights/proto";

message Options {
  int32 interval_in_seconds = 1;
  double latitude = 2;
  double longitude = 3;
  double radius = 4;
}

message Flight {
  double latitude = 1;
  double longitude = 2;
  string country = 3;
  string callSign = 4;
  string icao24 = 5;
  double velocity = 6;
}

service NearbyFlights {
  rpc Receive(stream Options) returns (stream Flight);
}
```

**Command line arguments / config**
```
.\ghz.exe --insecure --proto proto\service.proto --call proto.NearbyFlights.Receive -d '{"interval_in_seconds":10,"latitude":-23.43,"longitude":-46.45,"radius":5000}' localhost:8080
```

**Describe the bug**
When I run the command above I get the following error message:
```
ghz.exe: error: Error unmarshaling data '{interval_in_seconds:10,latitude:-23.43,longitude:-46.45,radius:5000}': invalid character 'i' looking for beginning of object key string
```

**To Reproduce**
Run the command above.

**Expected behavior**
I was expecting for this to work since the JSON is valid.

**Environment**
 - OS: Windows 10
 - ghz: latest
 - Go 1.15.3

**Additional context**
If I run the command like this:
```
.\ghz.exe --insecure --proto proto\service.proto --call proto.NearbyFlights.Receive -D example.json localhost:8080
```

And the JSON being like this:
```json
{
  "interval_in_seconds": 10,
  "latitude": -23.43,
  "longitude": -46.45,
  "radius": 5000
}
```
It works successfully.

**Comment by bojand** [comment]
Hmm I have not been able to reproduce this, although I am using macOS. I wonder if this is something with how Windows interprets the string argument in the command line?

**Comment by MoienTajik** [comment]
I have the same issue on Windows.

**Comment by fagnercarvalho** [comment]
TLDR: For anyone reading this issue, this is happening because of a limitation in Windows PowerShell and CMD.

To fix this escape the double quotes with backslashes like this:

```
'{\"interval_in_seconds\":10,\"latitude\":-23.43,\"longitude\":-46.45,\"radius\":5000}'
```

Investigation:

First I copied a code snippet of `ghz` where this issue was happening and I was able to reproduce the issue and I also noticed the double quotes were being stripped from the string:

```go
package main

import (
	"encoding/json"
	"fmt"
	"github.com/alecthomas/kingpin"
)

func main() {
	data := kingpin.Flag("data", "The call data as stringified JSON. If the value is '@' then the request contents are read from stdin.").
		Short('d').PlaceHolder(" ").String()

	kingpin.Parse()

	fmt.Println(*data)

	var dataObj interface{}
	if err := json.Unmarshal([]byte(*data), &dataObj); err != nil {
		fmt.Println(fmt.Errorf("error unmarshaling data '%v': %v", *data, err.Error()))
	}

	fmt.Println(dataObj)
}
```

Output:
```
{interval_in_seconds:10,latitude:-23.43,longitude:-46.45,radius:5000}
error unmarshaling data '{interval_in_seconds:10,latitude:-23.43,longitude:-46.45,radius:5000}': invalid character 'i' looking for beginning of object key string
<nil>
```

Then I took kingpin out of the equation by using `os.Args` and saw the same issue:
```go
package main

import (
	"encoding/json"
	"fmt"
	"os"
)

func main() {
	data := os.Args[1]

	fmt.Println(data)

	var dataObj interface{}
	if err := json.Unmarshal([]byte(data), &dataObj); err != nil {
		fmt.Println(fmt.Errorf("error unmarshaling data '%v': %v", data, err.Error()))
	}

	fmt.Println(dataObj)
}
```

However If I run any of these samples in WSL with Ubuntu I have no issues:
```
{"interval_in_seconds":10,"latitude":-23.43,"longitude":-46.45,"radius":5000}
map[interval_in_seconds:10 latitude:-23.43 longitude:-46.45 radius:5000]
```

Then I finally found this issue: https://github.com/golang/go/issues/30952. From there you will have links for issues in the PowerShell repo and there is some discussion involved with a lot of moving parts.

Closing this since it is not an issue in `ghz`
